framework/src/wifi_manager/wifi_manager_lwnl_listener.c

```diff
@@ -112,9 +112,7 @@ static int _lwnl_call_event(int fd, lwnl_cb_status status, int len)
 		LWNL_SET_MSG(&g_msg, WIFIMGR_EVT_STA_DISCONNECTED, WIFI_MANAGER_FAIL, NULL, NULL);
 		break;
 	case LWNL_EVT_SOFTAP_STA_JOINED:
-#ifdef CONFIG_WIFIMGR_DISABLE_DHCPS
```

Comment:
@j11591 Could you elaborate why this modification is necessary?

Reply:
In lwnl_fetch_event, (void)_lwnl_call_event(fd, status, len); is called.
g_msg is not set by not calling LWNL_SET_MSG, previous msg can be use for lwnl_fetch_event.